tests/test_aaa_authorization_service.rb

```diff
@@ -22,54 +22,72 @@ class TestAaaAuthorizationService < CiscoTestCase
   def setup
     super
     feature_tacacs
+    preconfig_tacacs_server_access(tacacs_groups[0])
```

Comment:
Can you just combine `preconfig_tacacs_server_access` and `config_tacacs_servers` into a single method?  What is the rational for separating this out?